speedy-recovery-client/src/components/secured/calendar/patient/PatientPractitionerCard.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import "react-big-calendar/lib/css/react-big-calendar.css";
+import "react-big-calendar/lib/addons/dragAndDrop/styles.less";
+import "../CalendarPages.css";
+import { Card, Image } from "semantic-ui-react";
+
+class PatientPractitionerCard extends React.Component {
+    
+    render() {
+        return (
+            <Card centered>
+                {this.props.selectedPractitioner !== undefined ?
+                    this.props.selectedPractitioner.photo !== undefined &&
+                    this.props.selectedPractitioner.photo !== "Unknown" ?
+                        <Image src={"data:image/png;base64," + this.props.selectedPractitioner.photo}/>
+                        : <Image src={require('../../../../defaultImages/maleDoctor.png')}/>
+                    : <Image src={require('../../../../defaultImages/femaleDoctor.jpg')}/>
+                }
+                <Card.Content>
```

Comment:
Can you please also hide this box when no practitioner is selected? It is confusing to have all the "data not foudn" when none is selcted